src/pages/events/[id]/public-images.tsx

```diff
@@ -53,23 +102,61 @@ export default function Images() {
           <>
             <Header {...headerValues} />
             <div className="container">
-              <div className="w-auto p-4 text-2xl font-bold text-zinc-400">
-                <span>Images ({Images().length})</span>
+              <div className="w-auto p-4 text-2xl font-bold text-app-base-200">
+                <span>Images ({images.length})</span>
               </div>
               <div className="container mx-auto">
                 <div className="flex justify-end"></div>
               </div>
```

Comment:
This div doesn't look it is used, can probably be removed.

Reply:
Thanks for spotting this!